CRM/Activity/Form/Search.php

```diff
@@ -195,7 +195,6 @@ public function postProcess() {
       $specialParams = array(
         'activity_type_id',
         'status_id',
-        'activity_subject',
```

Comment:
you can replace it with activity_text``` as ```CRM_Contact_BAO_Query::processSpecialFormValue(...)``` can format LIKAble attributes here https://github.com/twomice/civicrm-core/blob/96e30da363f70bb7919276a87567892cfad0d60b/CRM/Contact/BAO/Query.php#L6073 and eventually it optimise ```whereClauseSingleActivityText(...)``` to handle all kind of operator not just hardcoded LIKE